maro/rl/policy/policy.py

```diff
@@ -60,6 +62,19 @@ def get_rollout_info(self):
     def get_batch_loss(self, batch: dict, explicit_grad: bool = False):
         pass
 
+    def data_parallel(self, *args, **kwargs):
+        self.remote = True
+        self._proxy = Proxy(*args, **kwargs)
+
+    def data_parallel_with_existing_proxy(self, proxy):
+        self.remote = True
+        self._proxy = proxy
+
+    def exit_data_parallel(self):
+        self.remote = False
+        if hasattr(self, '_proxy'):
+            self._proxy.close()
+
```

Comment:
Do we need a ``learn_with_data_parallel`` interface for the abs RLPolicy class?

Reply:
It could have.